addons/sys_gui/XEH_postInit.sqf

```diff
@@ -41,9 +42,37 @@ DFUNC(enterVehicle) = {
     } else {
         // Close vehicle info display
         (QGVAR(vehicleInfo) call BIS_fnc_rscLayer) cutText ["", "PLAIN"];
+        //(QGVAR(antennaElevationInfo) call BIS_fnc_rscLayer) cutRsc [QGVAR(antennaElevationInfo), "PLAIN", 0, false];
     };
 };
 
+DFUNC(antennaElevationDisplay) = {
```

Comment:
This should be a separate function file.

Reply:
Done